tests/conftest.py

```diff
@@ -157,8 +157,11 @@ def mxnet_training_py_version(mxnet_training_version, request):
 
 
 @pytest.fixture(scope="module", params=["py2", "py3"])
-def mxnet_eia_py_version(request):
-    return request.param
+def mxnet_eia_py_version(mxnet_eia_version, request):
+    if Version(mxnet_eia_version) < Version("1.7.0"):
+        return request.param
+    else:
+        return "py3"
```

Comment:
I believe this might need to be py36 as per https://github.com/aws/deep-learning-containers/commit/cab6c4e2d11aaf27524d56732aa37921d5308895.